### DISCUSSION TITLE
Saving imported beatmap/skin data on the osu root folder

### DISCUSSION THREAD

**Discussion by farsdewibs0n** [ppy/osu#16219] [Q&A] [answered]
I saved my osu!lazer folder on D: Drive, when I checked WinDirStat, there are 30GB worth of osu data on my Local fodler (where the default installed location). When I delete it, turns out it contains maps and skins data. Is there a way to move the folders to the root osu folder?

**Comment by bdach** [comment, marked as answer]
i don't know what this means: 

> Is there a way to move the folders to the root osu folder?

can you explain further what you're trying to do?

if the files on D:/ drive are indeed used, there should be a `storage.ini` file in `%AppData%/osu`, pointing at the data directory on D:/.

**Reply by farsdewibs0n** [reply]
I put lazer on my D: Drive. when I import songs and skins from stable, it fills up my C: drive (located in the Local folder) instead of the lazer installation folder (In which case, D:).

**Reply by bdach** [reply]
you used the in-game location switcher, correct?

if yes, then i'd like a proper answer to this:

> if the files on D:/ drive are indeed used, there should be a `storage.ini` file in `%AppData%/osu`, pointing at the data directory on D:/.

is there such a file?

**Reply by farsdewibs0n** [reply]
> you used the in-game location switcher, correct

What is that? idk this is a thing


> is there such a file?

found it

**Reply by bdach** [reply]
> What is that? idk this is a thing

![2021-12-28-091227_625x341_scrot](https://user-images.githubusercontent.com/20418176/147544234-a670b4a3-e21d-479f-a1fb-5bfd452fbec9.png)

did you use this?

> found it

well, what's in the file?
